tests/ui/runtime/running-with-no-runtime.rs

```diff
@@ -2,15 +2,15 @@
 //@ ignore-wasm32 spawning processes is not supported
 //@ ignore-sgx no processes
 
-#![feature(start)]
+#![no_main]
 
 use std::ffi::CStr;
 use std::process::{Command, Output};
 use std::panic;
 use std::str;
 
-#[start]
-fn start(argc: isize, argv: *const *const u8) -> isize {
+#[no_mangle]
+extern "C" fn main(argc: isize, argv: *const *const u8) -> isize {
```

Comment:
These technically have to use `c_int` instead of `isize` as the C side uses `int` as types for `main`.

Reply:
Ah, good point. Both argc and the return type have to be c_int, right?